src/main/webapp/css/map.css

```diff
@@ -0,0 +1,14 @@
+body {
+	background-color: lightblue;
```

Comment:
In order to keep things consistent between pages, you should just include main.css in your HTML file and just use the body CSS in there instead of adding your own here.